specification/app/resource-manager/Microsoft.App/stable/2024-03-01/Subscriptions.json

```diff
@@ -0,0 +1,80 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-03-01",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/getCustomDomainVerificationId": {
```

Comment:
>getCustomDomainVerificationId [](http://example.com/codeflow?start=61&length=29)

in ARM we prefer to model everything as resource asmuch as possible.

Can you please instead model this as a readonly resource instead of making this a POST call ?

Reply:
it is a Post call in our backend and already be there in previous api version 2023-05-02-preview